### DISCUSSION TITLE
Mobile Application of Stellarium

### DISCUSSION THREAD

**Discussion by abdeldjalil-fellah** [Stellarium/stellarium#383] [Q&A] [answered]
Do you plan to release a mobile version of Stellarium (Android, iOS)? 
Is there any alternative one else?

**Comment by gzotti** [comment, marked as answer]
There is a mobile version. And there are alternatives. 

See https://github.com/Stellarium/stellarium/pull/194 for another attempt. It requires a large screen though. Making it usable on regular 5.* inch phone screens common today likely requires developing a new GUI. Also some more touch screen things (multitouch etc.) would be required to make it more useful.

In case you are experienced Android developer, feel free to provide a working solution for (optimally) Android 4, regular phone screen size, and a very good GUI. Of course, don't mess up the PC GUI...
